libs/ui/map/src/lib/components/map-legend/map-legend.component.css

```diff
@@ -0,0 +1,3 @@
+.geosdk--legend-container {
+  overflow: auto;
```

Comment:
How about also making it so that the layer label wraps on several lines instead of causing a scrollbar to appear?